src/backend/base/langflow/base/flow_processing/utils.py

```diff
@@ -62,7 +62,7 @@ def build_records_from_result_data(result_data: ResultData, get_final_results_on
         else:
             return []
 
-    for message in messages:
+    for message in messages:  # type: ignore
```

Comment:
Assuming this complains because `messages` is optional. Could add `if messages: ...`